demo.html

```diff
@@ -128,14 +129,70 @@ <h2 class = "section-title">Lead body copy</h2>
         </div>
 
         <div class="sample-code">
-          <p>
-            <span class="element-color">&lt;p</span>
-            <span class="class">class=</span><!--
-         --><span class="class-name">"lead"</span><!--
+          <code>
+            <p>
+              <span class="element-color">&lt;p</span>
+              <span class="class">class=</span><!--
+           --><span class="class-name">"lead"</span><!--
+           --><span class="element-color">&gt;</span><!--
+           -->...<!--
+           --><span class="element-color">&lt;/p&gt;</span>
+            </p>
+          </code>
+        </div>
+      </header>
+
+      <header id="address">
+        <h1 class="section-title">Addresses</h1>
+
+        <div class="breakdown">
+          <p>Present contact information in our default style by using the <code class="element-in-text">&lt;address&gt;</code> tag. Preserve your formatting by ending all lines with <code class="element-in-text">&lt;br&gt;</code>.</p>
+        </div>
+
+        <div class="sample-text">
+          <address>
+            <strong>Learners Guild</strong><br>
+            492 Ninth Street (Garden Level)<br>
+            Oakland, CA 94607<br>
+            <abbr title="Phone">P:</abbr> (123) 456-7890
+          </address>
+
+          <address>
+            <strong>Full Name</strong><br>
+            <a href="mailto:#">first.last@example.com</a>
+          </address>
+        </div>
+
+        <div class="sample-code">
+          <code>
+            <span class="element-color">&lt;address&gt;</span><br>
+            <span class="tab element-color">&lt;strong&gt;</span><!--
+         -->Learners Guild<!--
```

Comment:
This is inline text and the spans create extra spaces we don't want, the comments around the line breaks in our code get rid of those spaces.